src/schema.ts

```diff
@@ -1199,6 +1207,14 @@ export const typeDefs = gql`
     groupTitleArray: [String!]!
   }
 
+  type ArenaUserNotification implements BaseUser {
```

Comment:
Det blir ikkje litt meir lesbart slik?
```suggestion
  type ArenaNotificationUser implements BaseUser {
```

Reply:
Jo!